spec/Section 7 -- Response.md

```diff
@@ -41,6 +59,11 @@ requested operation. If the operation was a query, this output will be an
 object of the query root operation type; if the operation was a
 mutation, this output will be an object of the mutation root operation type.
 
+If the result of the operation is an event stream, the `data` entry in 
```

Comment:
This might be a good place to explicitly allow the event stream to contain response maps without data or errors, but with extensions, to allow for communication of individual stream iterator closure via extensions (assuming it will not be directly included within the spec).

Reply:
I updated the extensions section to note that payloads with only `hasNext` and `extensions` are allowed.